Jenkinsfile

```diff
@@ -1,7 +1,3 @@
 #!/usr/bin/env groovy
 
 library("govuk")
-
-node {
-  govuk.buildProject(publishingE2ETests: true, rubyLintDiff: false)
-}
```

Comment:
I think you've removed a bit much here 😆  https://ci.integration.publishing.service.gov.uk/job/email-alert-service/job/remove-docker/1/console - buildProject isn't implicitly run

Reply:
Oops - I forgot this is Go and not some weird configuration language.

Fixed 👍